json-schemas/src/client-events-api-requests.json

```diff
@@ -0,0 +1,41 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "Ably API Requests Client Event",
+  "type": "object",
+  "properties": {
+    "host": {
+      "type": "string",
+      "description": "the host the request was made to"
+    },
+    "requestId": {
+      "type": "string",
+      "description": "the unique ID of the request that Ably can use to correlate a request event with internal logs, if necessary"
+    },
+    "region": {
+      "type": "string",
+      "description": "the region the request was made in"
+    },
+    "headers": {
+      "type": "object",
+      "description": "the headers sent with the request"
```

Comment:
```
The HTTP headers of the API request.
```

Reply:
Also, it's worth noting that the HTTP header keys are normalised to all lower case (i.e. `user-agent` rather than `User-Agent`)

Reply:
Yes. Do you mean that should be included in the description? Or that the code should enforce that by making all headers lower case?

Reply:
I think it should be in the description to inform the user, but no enforcement necessary since HTTP header names are case insensitive.